README.md

```diff
@@ -241,12 +241,12 @@ seen in the screenshot below.
 #### VideoView in a Layout File 
 
     <?xml version="1.0" encoding="utf-8"?>
-    <FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
-                 xmlns:app="http://schemas.android.com/apk/res-auto"
-                 android:id="@+id/video_container"
-                 android:layout_width="match_parent"
-                 android:layout_height="match_parent"
-                 android:keepScreenOn="true">
+    <merge xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
```suggestion
    <FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
This may have fixed a lint error in our own quickstart, but I think we should keep this as a FrameLayout in the documentation as developers may have a different parent layout such as a ConstraintLayout, which would result in a runtime crash.